src/store/store.js

```diff
@@ -11,11 +11,7 @@ const configureStore = () => {
   const sagaMiddleware = createSagaMiddleware();
   const store = createStore(
     rootReducer(history),
-    compose(
-      applyMiddleware(routerMiddleware(history), sagaMiddleware),
-      window.__REDUX_DEVTOOLS_EXTENSION__ &&
```

Comment:
Please, revert this change because we need redux devtools in development

Reply:
Done